### ISSUE TITLE
Software Center styling issues

### ISSUE THREAD

**Issue by madsrh** [ubuntu/yaru#895]
A three minor styling issues with the new version in Cosmic: 
These are bug fixes 😉 

### There's now a Add-ons part of the installed list:
Perhaps just add a light gray background and continue the left+right borders
* [Current](https://user-images.githubusercontent.com/3986894/46563789-c612b600-c903-11e8-92df-89bec969f90a.png)
* [Mockup](https://user-images.githubusercontent.com/3986894/46563852-243f9900-c904-11e8-9983-e38d15fcd831.png)

### When searching for an app, the list is missing bottom border:
![image](https://user-images.githubusercontent.com/3986894/46563894-5224dd80-c904-11e8-90e0-b3565a9fd70e.png)

### Border for updates is too thick:
![image](https://user-images.githubusercontent.com/3986894/46563982-edb64e00-c904-11e8-990c-a378b77f4765.png)

**Comment by madsrh** [comment]
@Feichtmeier @clobrano Could you add this? 👍 (ofc not _#cdcdcd_ but the SCSS variable)


```
.app-listbox-header { background: #cdcdcd; }

```


![image](https://user-images.githubusercontent.com/3986894/46581403-65d16080-ca38-11e8-85c0-9b7669e83892.png)

## Not that much success
The update list has a dobbelt border (as seen above), but I have no idea how to work around this. I tried something like this, but it introduces other issues:

```
.app-listbox-header { background: #cdcdcd; }
.app-updates-section { border-color: #cdcdcd; border-left: none; border-right: none; border-radius: 0px; }
.list-box-app-row { border-bottom: 2px solid #cdcdcd; }
.activatable { border-bottom: none; }
```

![image](https://user-images.githubusercontent.com/3986894/46581444-353df680-ca39-11e8-9af2-7f8f67ad13d4.png)

![image](https://user-images.githubusercontent.com/3986894/46581449-4850c680-ca39-11e8-9c17-ec30abe840fa.png)

**Comment by clobrano** [comment]
About missing bottom border: the fix would very likely requires a style change. The bottom border is eaten by a *negative margin* needed to show the applications a a continuous list of rows, while it is actually a list of boxes with a separator in between, that you can easily see it between builder and chromium in below picture.

![image](https://user-images.githubusercontent.com/2883614/46610044-e14f1280-cb09-11e8-92b1-8cc2689ec329.png)

I am trying to make the separator invisible

**Comment by clobrano** [comment]
@madsrh what do you think about this?

![image](https://user-images.githubusercontent.com/2883614/46611894-64bf3280-cb0f-11e8-94b4-dea4128da781.png)

**Comment by madsrh** [comment]
That is a solution 👍 
I think that the current master is less cluttered because all thoughs lines - look kind of messy here. So +1 if the single line isn't possible 😄

**Comment by clobrano** [comment]
I made it :D 

![image](https://user-images.githubusercontent.com/2883614/46617073-2c722100-cb1c-11e8-9172-f802820e228a.png)

![image](https://user-images.githubusercontent.com/2883614/46617043-19f7e780-cb1c-11e8-9286-e945ea0fe658.png)

Could you try the PR for the add-ons header? My software center does not show it
Finally, I shall wait for updates to fix the last part :D

**Comment by clobrano** [comment]
@madsrh the app-list-box-header is used also in this header

![image](https://user-images.githubusercontent.com/2883614/46655631-77d41000-cbac-11e8-8d49-f32ad4554863.png)

**Comment by madsrh** [comment]
@clobrano Yes, it should have the gray background too. That will look good.

**Comment by madsrh** [comment]
Uhh, that default font-size 🤢 Surprises me every time.

Anyway, **+1** LGTM
